LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Roman Shtylman
```

Comment:
Ah, no, this probably needs to be kept as Roman.  It's still largely his copyright.